doc/test-framework/scorecard.md

```diff
@@ -79,29 +84,245 @@ API server, indicating that it is modifying resources. This test has a maximum s
 
 #### Provided APIs have validation
 
-This test verifies that all the CRDs in the CRDs folder contain a validation section. If the CRD matches the kind and version of the
-CR currently being tested, it will also verify that there is a validation for each spec and status field in that CR. This test has a
-maximum score of 1.
+This test verifies that the CRDs for the provided CRs contain a validation section and that there is a validation for each
```

Comment:
```suggestion
This test verifies that the CRDs for the provided CRs contain a validation section and that there is validation for each
```

The way validation in relation to CRDs is referred to is a bit wonky in general, but this reads the best IMO.